tests/plugins/test_pull_base_image.py

```diff
@@ -190,6 +190,15 @@ def test_pull_base_wrong_registry(reactor_config_map):  # noqa
                                     reactor_config_map=reactor_config_map)
 
 
+# test previous issue https://github.com/projectatomic/atomic-reactor/issues/1008
+def test_pull_base_library(reactor_config_map):  # noqa
+    with pytest.raises(PluginFailedException) as exc:
+        test_pull_base_image_plugin(
+            LOCALHOST_REGISTRY, "spam/library-only:latest", [], [], reactor_config_map
+        )
+    assert "not found" in str(exc.value)
```

Comment:
this should be 'not in' as it should raise error as namespace is defined

but the raise error actually raises : 
2018-06-04 21:20:21,354 platform:- - atomic_reactor.plugin - ERROR - plugin 'pull_base_image' raised an exception: RetryGeneratorException("Failed to mock_method image (u'localhost:5000/spam/library-only',): u'Error: image ***:latest not found'",)